pkg/git/libgit2/checkout_test.go

```diff
@@ -562,3 +710,9 @@ func TestCheckoutStrategyForOptions(t *testing.T) {
 		})
 	}
 }
+
+func enableManagedTransport() {
+	fg := feathelper.FeatureGates{}
+	fg.SupportedFeatures(features.FeatureGates())
+	managed.InitManagedTransport(logr.Discard())
+}
```

Comment:
As this is not being called in this file, would it make sense to keep this elsewhere? Maybe on `managed_test.go`?